Library/src/Web/EmscriptenWebClient/EmscriptenWebClient.h

```diff
@@ -15,6 +15,10 @@
  */
 #pragma once
 
+#include "CSP/CSPCommon.h"
+
+//#ifdef CSP_WASM
```

Comment:
Remove please! Also, the header above isn't needed if this symbol check is removed.